src/packageUtils.js

```diff
@@ -30,6 +32,58 @@ const getNpmConfig = _.memoize(() => {
   return config;
 });
 
+export function createGlobalPackageJson() {
+  // retrieve the global package install path
+  const res = shell.exec('npm root -g', {silent: true});
```

Comment:
Why not use `npm ls -g --depth 0 --json` to get list of global packages with their versions?